app/controllers/application_controller.rb

```diff
@@ -21,4 +21,14 @@ def error500(e)
     logger.error [e, *e.backtrace].join("\n")
     render 'error500', status: 500, formats: [:html]
   end
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+  protected
```

Comment:
__[rubocop]__
* Style/EmptyLinesAroundAccessModifier: Keep a blank line before and after `protected`.